test/4.public.UrlChecker.js

```diff
@@ -139,9 +139,83 @@ describe("PUBLIC -- UrlChecker", function()
 			}).enqueue( conn.absoluteUrl );
 		});
 	});
-	
-	
-	
+
+
+
+	describe("caching", function()
+	{
+		var options;
+		beforeEach(function()
+		{
+			options = utils.options();
+			options.cacheResponses = true;
+		});
+
+		it("should check unique url only once", function(done)
+		{
+			var results = [];
+			
+
+			var instance = new UrlChecker( options,
+			{
+				link: function(result, customData)
+				{
+					results.push(result);
```

Comment:
Okay, I just stole this logic from the other tests. Would you like those updated as well or that change to be done in a different PR?